src/ripple/app/misc/SHAMapStoreImp.cpp

```diff
@@ -181,19 +181,22 @@ SHAMapStoreImp::SHAMapStoreImp (
     , nodeStoreJournal_ (nodeStoreJournal)
     , transactionMaster_ (transactionMaster)
     , canDelete_ (std::numeric_limits <LedgerIndex>::max())
+    , rotating_(false)
```

Comment:
gcc warns about initialization order here. `rotating_` needs to be moved to before `transactionMaster_` to match the actual order of initialization